front/app/modules/commercial/customizable_homepage_banner/citizen/TwoColumnLayout.tsx

```diff
@@ -8,9 +8,9 @@ import Image from 'components/UI/Image';
 import { homepageBannerLayoutHeights } from 'containers/Admin/settings/customize/Header/HeaderImageDropzone';
 
 const Container = styled.div`
+  width: 100%;
```

Comment:
Was also something wrong with the two-column  layout. It had spacing on the left/right.